src/GoogleStorageAdapter.php

```diff
@@ -161,19 +162,18 @@ public function rename($path, $newpath)
      */
     public function copy($path, $newpath)
     {
-        $params = [
-            'destinationPredefinedAcl' => $this->getRawVisibility($path) == AdapterInterface::VISIBILITY_PUBLIC ?
-                'publicRead' :
-                'private'
-        ];
+        $originalVisibility = $this->getRawVisibility($path);
+
         $this->service->objects->copy(
             $this->bucket,
             $path,
             $this->bucket,
             $newpath,
-            new \Google_Service_Storage_StorageObject(),
-            $params
+            new \Google_Service_Storage_StorageObject()
         );
+
+        $originalVisibility === AdapterInterface::VISIBILITY_PUBLIC ? $this->publishObject($newpath) : $this->unPublishObject($newpath);
```

Comment:
Here this code uses strict comparisons and a ternary to call the functions and then later on [here](https://github.com/Superbalist/flysystem-google-storage/pull/4/files#diff-04e077be29e970a3aa87de2a2abd4f68R221), we use an if-else. 

I think we should use strict comparisons everywhere, if possible (even if we aren't already), and then call functions in an if-else, because it's easier to read.